samples/Storage/https-traffic-only/azurepolicy.json

```diff
@@ -12,10 +12,36 @@
                         "equals": "Microsoft.Storage/storageAccounts"
                     },
                     {
-                        "not": {
-                            "field": "Microsoft.Storage/storageAccounts/supportsHttpsTrafficOnly",
-                            "equals": "true"
-                        }
+                        "anyOf": [
+                            {
+                                "allOf": [
+                                    {
+                                        "value": "[requestContext().apiVersion]",
+                                        "in": [
```

Comment:
can you use "less": "2019-04-01" instead so this will support other environments that might not have different older api-versions?

Reply:
I didn't know less works with api-versions. Nice!

Reply:
would it still work with versions that have -preview at the end?